src/metabase/models/dashboard_card.clj

```diff
@@ -97,10 +97,11 @@
   :series
   "Return the `Cards` associated as additional series on this DashboardCard."
   [{:keys [id]}]
-  (db/select [Card :id :name :description :display :dataset_query :visualization_settings :collection_id]
-    (mdb.u/join [Card :id] [DashboardCardSeries :card_id])
-    (db/qualify DashboardCardSeries :dashboardcard_id) id
-    {:order-by [[(db/qualify DashboardCardSeries :position) :asc]]}))
+  (t2/select [Card :id :name :description :display :dataset_query :visualization_settings :collection_id]
```

Comment:
the only place that changes in syntax, all other places are just simple string replacements.